clouddriver-lambda/src/main/java/com/netflix/spinnaker/clouddriver/lambda/deploy/ops/UpsertLambdaEventSourceAtomicOperation.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2018 Amazon.com, Inc. or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.lambda.deploy.ops;
+
+import com.amazonaws.services.lambda.AWSLambda;
+import com.amazonaws.services.lambda.model.*;
+import com.netflix.spinnaker.clouddriver.lambda.cache.model.AwsLambdaCacheModel;
+import com.netflix.spinnaker.clouddriver.lambda.deploy.description.UpsertLambdaEventMappingDescription;
+import com.netflix.spinnaker.clouddriver.orchestration.AtomicOperation;
+
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class UpsertLambdaEventSourceAtomicOperation extends AbstractAwsLambdaAtomicOperation<UpsertLambdaEventMappingDescription, Object> implements AtomicOperation<Object> {
+
+
+  public UpsertLambdaEventSourceAtomicOperation(UpsertLambdaEventMappingDescription description) {
+    super(description, "UPSERT_LAMBDA_FUNCTION_EVENT_MAPPING");
+  }
+
+  @Override
+  public Object operate(List priorOutputs) {
```

Comment:
I don't think I've seen that pattern of returning `Object` anywhere else in clouddriver, is there any way to be more specific?